src/container/content/search/search.js

```diff
@@ -2,6 +2,9 @@ import React from 'react';
 import '../search/search.css';
 import '../search/media.css';
 import rozdil from '..//..//..//image/rozdil_framed.png';
+import middleCircle from '..//../../image/middle_circle.png';
```

Comment:
@Babiy7 There are no images files, please add them and push
./src/container/content/alphabet/alphabet.js
Module not found: Can't resolve '..//../../image/double_circle.png' in '/Users/bogdan/Developer/KeyToTech/KoreanPhraseBookSite/src/container/content/alphabet'